.all-contributorsrc

```diff
@@ -256,6 +256,16 @@
       ]
     },
     {
+      "login": "Axfez",
+      "name": "Santiago Montoya Rendón",
+      "avatar_url": "https://avatars.githubusercontent.com/u/92069466?v=4",
+      "profile": "https://github.com/Axfez",
+      "contributions": [
+        "question",
+        "code",
+        "data",
+        "doc",
+        "maintenance"
```

Comment:
You're missing a closing bracket `]` and curly brace `}`

Reply:
Fixed